src/V3AstNodeOther.h

```diff
@@ -2120,7 +2122,7 @@ class AstVar final : public AstNode {
     void setIgnoreSchedWrite() { m_ignoreSchedWrite = true; }
     bool dfgMultidriven() const { return m_dfgMultidriven; }
     void setDfgMultidriven() { m_dfgMultidriven = true; }
-
+    void setGlobalConstrained(bool flag) { m_globalConstrained = flag; }
```

Comment:
Convention for setter is if it can be set to an arbitrary value, function name should be same as member name minus the `m_` prefix. If it can only be set true, but never cleared, then use `set` prefix and take no value argument.